repos/base-linux/src/core/spec/linux/io_mem_session_component.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ * \brief  Linux-specific IO_MEM service
+ * \author Christian Helmuth
```

Comment:
Nope, I did not create this file ;-)

Reply:
Iirc this is also the case for `repos/base-linux/src/core/spec/pc/io_mem_session_component.cc` ;)